### ISSUE TITLE
Browser extension for Accessibility Insights icon is not visible in high contrast "High Contrast Black" mode

### ISSUE THREAD

**Issue by sahithikkss-zz** [microsoft/accessibility-insights-web#4516]
## Describe the bug

Browser extension for Accessibility Insights icon is not visible in high contrast "High Contrast Black" mode

**To Reproduce**

1. Go to Edge browser with Accessibility Insights installed
2. Change Windows High contrast settings to "High Contrast Black" 
3. Icon visibility is less and does not meet contrast ratio

## Expected behavior
Clear visibility of Accessibility Insights Extension Icon in all high contrast modes

## Screenshots
![image](https://user-images.githubusercontent.com/53164851/126674843-368e7348-b6b1-4be5-b7f0-1d1bf5eb1c84.png)

![image](https://user-images.githubusercontent.com/53164851/126675469-e6279c43-faaa-49bb-bb61-e036b69a69ed.png)
## Context (please complete the following information)

- OS Name & Version: 
- AI-Web Version & Environment: 
- Browser Version: Edge/Chrome 
- Target Page: 

## Are you willing to submit a PR?
No

## Did you search for similar existing issues?
No

## Additional context
N/A

**Comment by AhmedAbdoOrtiga** [comment]
I see the issue but I think it needs more investigation to see if we can can programmatically change extension icon when HC mode is on/off

**Comment by DaveTryon** [comment]
@ferBonnin, our solution in the Windows app was to request new icons that include an internally contrasting border, so that we have correct contrast no matter what the background color happens to be. The same approach might also be viable in the extension.

**Comment by ferBonnin** [comment]
@DaveTryon thanks for sharing! this might be a good option instead of having to change the icons' colors

**Comment by ferBonnin** [comment]
per conversation with @peterdur lets try to make the background of the icon a white circle. Please let us know if a new image from design is needed.
![the Accessibility insights icon inside a white circle](https://user-images.githubusercontent.com/9062104/127059941-fc4c94f4-5ea4-41a1-9abb-22c94c789757.png)

**Comment by peterdur** [comment]
@ferBonnin Looking into the images, we'll need new versions of:
- https://github.com/microsoft/accessibility-insights-web/blob/main/src/icons/brand/blue/brand-blue-16px.png
- https://github.com/microsoft/accessibility-insights-web/blob/main/src/icons/brand/blue/brand-blue-48px.png
- https://github.com/microsoft/accessibility-insights-web/blob/main/src/icons/brand/blue/brand-blue-128px.png

Something to think about: The heart icon in the existing images approaches the edges of the image (the grey box in the image below); to add the circle would require making the heart much smaller.
![image](https://user-images.githubusercontent.com/1752950/133175830-6f828e50-3339-456c-b052-121995308a37.png)
We may want to consider adding a white border to the heart instead.

**Comment by ferBonnin** [comment]
@peterdur got it, let me check with design.

**Comment by ferBonnin** [comment]
@peterdur, we met with the design team. I am attaching a file with the 3 sizes and an 8px white border. Ok to mark this as _ready for work_? 

[new icons.zip](https://github.com/microsoft/accessibility-insights-web/files/7247285/new.icons.zip)

**Comment by peterdur** [comment]
> we met with the design team. I am attaching a file with the 3 sizes and an 8px white border. Ok to mark this as _ready for work_?

@ferBonnin Looks like we need some tweaks; there's no 16px icon, and the 128px icon is a white heart rather than blue (or is that expected?)

**Comment by ferBonnin** [comment]
> @ferBonnin Looks like we need some tweaks; there's no 16px icon, and the 128px icon is a white heart rather than blue (or is that expected?)
Ops, my bad! all icons should be blue. I will request a 16px icon, doesn't look like we have it in the Figma yet

**Comment by ferBonnin** [comment]
updated the icons file above

**Comment by ferBonnin** [comment]
@peterdur I think this can now be moved to Ready for work?

**Comment by peterdur** [comment]
Holding this for a warm-up project

**Comment by ferBonnin** [comment]
This has been resolved in the Production icon.
